tests/integration_tests/performance/goldfile.json

```diff
@@ -0,0 +1,15 @@
+{
+    "network_latency": {
+        "description" : "Average network latency for a Firecracker VM. The results are round-trip.",
+        "x86_64" : {
+            "target" : 0.140,
```

Comment:
What units? Can we mention the units in the description?

Reply:
Good idea. I will update this

Reply:
Just to double check: we have here 0.140 milliseconds or 140 milliseconds?

Reply:
These are 0.140 milliseconds

Reply:
Cool. Thanks.